### PR TITLE
Clean up error.log warnings

### PR DESCRIPTION
**Pull request by Grociu**
Cleans up almost everything that pops up in the error.log file
Duplicate missions had their names changed
Added similar/equivalent paradox dds files as missing icons
Added missing mandatory elements to events
Commented out unused government types and trade goods references

### CODE REVIEW

File: events/ME_Rum_Events.txt [hunk 1]
```diff
@@ -30,19 +30,19 @@ country_event = {
...
+		#		TRA = {
+		#			NOT = { religion_group = muslim }
+		#		}
+		#	}
+		#	create_subject = {
+		#		subject = TRA
+		#		subject_type = muslim_tributary_vassal
+		#	}
+		#}
 		else_if = {
```
Comment (stackpoint) [inline events/ME_Rum_Events.txt]: should be if

**Review by stackpoint — CHANGES_REQUESTED** [review]

File: events/ME_Rum_Events.txt (hunk 1 above)
Comment (Grociu) [reply events/ME_Rum_Events.txt]: There is an `if` above the commented section.

Comment (stackpoint) [reply events/ME_Rum_Events.txt:46]: Right but this else_if shouldn't be nested into that one.

Comment (stackpoint) [reply events/ME_Rum_Events.txt]: Right but this else_if shouldn't be nested into that one.

Comment (Grociu) [reply events/ME_Rum_Events.txt]: Changed.

**Review by stackpoint — APPROVED** [review]

**Comment by stackpoint** [comment]
Go ahead and merge after you resolve the conflict.

**Pull request merged**